SimPPS/RPDigiProducer/plugins/RPLinearChargeCollectionDrifter.cc

```diff
@@ -0,0 +1,33 @@
+#include "SimPPS/RPDigiProducer/interface/RPLinearChargeCollectionDrifter.h"
+#include "Geometry/VeryForwardRPTopology/interface/RPTopology.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include <iostream>
+#include <vector>
+
+RPLinearChargeCollectionDrifter::RPLinearChargeCollectionDrifter
+      (const edm::ParameterSet &params, RPDetId det_id)
+{
+  verbosity_ = params.getParameter<int>("RPVerbosity");
+  GeV_per_electron_ = params.getParameter<double>("RPGeVPerElectron");
+  charge_cloud_sigmas_vect_ = params.getParameter< std::vector<double> >("RPInterStripSmearing");
+  det_thickness_ = RPTopology().DetThickness();
+  _det_id=det_id;
+}
+
+simRP::charge_induced_on_surface RPLinearChargeCollectionDrifter::Drift
+      (const simRP::energy_path_distribution &energy_deposition)
+{
+  _temp.resize(energy_deposition.size());
```

Comment:
`_temp` seems to be used only in this function, in which case it would be better be as a local variable.